buildkite/data_source_ip_ranges_test.go

```diff
@@ -0,0 +1,64 @@
+package buildkite
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+// Confirm that we can create a new agent token, and then delete it without error
+func TestAccDataIpRanges_read(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataIpRangesConfigBasic(),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					// Confirm the ip ranges data source has the correct values in terraform state
+					testAccBuildkiteIPRangesCheckWebhookIps("data.buildkite_ip_ranges.foobar"),
+				),
+			},
+		},
+	})
+}
+
+func testAccBuildkiteIPRangesCheckWebhookIps(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		resourceState, ok := s.RootModule().Resources[resourceName]
+		attributes := resourceState.Primary.Attributes
+
+		if !ok {
+			return fmt.Errorf("Not found in state: %s", resourceName)
+		}
+
+		if resourceState.Primary.ID == "" {
+			return fmt.Errorf("No ID is set in state")
+		}
+
+		var (
+			webhookIpsSize int
+			err            error
+		)
+		if webhookIpsSize, err = strconv.Atoi(attributes["webhook_ips.#"]); err != nil {
+			return err
+		}
+
+		if webhookIpsSize == 0 {
+			return fmt.Errorf("webhook_ips attribute should have more than 0 items (len: %d)", webhookIpsSize)
+		}
+
+		return nil
+	}
+}
```

Comment:
I don't want the test to fail if we change the IPs returned by the API, so I'm just doing some basic sanity checks:

1. is there an ID set
2. are there 1 or more strings returned